x/nft/blockchain/codec.proto

```diff
@@ -16,12 +16,25 @@ message TokenDetails {
     IOV iov = 2 [(gogoproto.nullable) = false];
 }
 
+// Chain represents a blockchain information.
+//
+// Name chain is used instead of blockchain to not repeat the same word
+// everywhere.
 message Chain {
+    // Chain ID is the blockchain ID as referred in our system.
     string chain_id = 1 [(gogoproto.customname) = "ChainID"];
+
+    // TODO(piotr) what is this values used for?
```

Comment:
What is the `network_id` value for?

Reply:
https://iov-one.github.io/bns-spec/_build/html/docs/data/ObjectDefinitions.html#blockchain-registration-tokens
you can find this info here

Reply:
Thanks!

I still don't fully understand what it is though :sweat_smile: 

> The networkId is a field used by Ethereum and its derivatives, so this is also included as an option for chain configuration.

I understand it is used only by some blockchain implementations, like Ethereum, but what for? What should I write in the documentation?